RecoJets/JetProducers/plugins/MVAJetPuIdProducer.h

```diff
@@ -0,0 +1,48 @@
+#include "FWCore/Framework/interface/MakerMacros.h"
```

Comment:
maybe I was unclear: all of the MVAJetPuIdProducer.h content should be moved to MVAJetPuIdProducer.cc
Previous comments to this file were not for single lines in the file, but for all of the file.
Please follow up.
Thank you.